README.md

```diff
@@ -1,32 +1,42 @@
 # Unloop
 
 ## Technologies
-* Ruby 2.5.6
-* Rails 6.0.0
-* Postgresql 11.6
+
+- Ruby 2.5.6
+- Rails 6.0.0
+- Postgresql 11.6
 
 ## Installation
+
 ### Local Development
+
 1. Clone the repo
+
 ```bash
 git clone https://github.com/calblueprint/unloop.git
 cd unloop
 ```
+
 2. Install dependencies
+
 ```bash
 bundle install
 yarn install
 ```
+
 3. Setup database
+
 ```bash
+cp config/database.yml.example config/database.yml
```

Comment:
Could you please provide context for this command -- is this specific to only developing on a local machine? What should a developer keep in mind when they are deploying to production based on this file?

Reply:
by default ppl should use `database.yml.example` unless it doesn't work on their local machine (i.e. windows users)
i believe it should be run when deploying to production? there is a way to run the command before deployment. not sure how it's being done right now.

Reply:
Now we just create a new database.yml file to init the database.